.gitignore

```diff
@@ -87,3 +104,6 @@ ENV/
 
 # Rope project settings
 .ropeproject
+
+
+migrations/
```

Comment:
You probably want to track the migrations too, to make it easy to sync or rollback updates to your db models.